tests/components/lcn/test_websocket.py

```diff
@@ -0,0 +1,305 @@
+"""LCN Websocket Tests."""
+
+from pypck.lcn_addr import LcnAddr
+import pytest
+
+from homeassistant.components.lcn.const import CONF_DOMAIN_DATA
+from homeassistant.components.lcn.helpers import get_device_config, get_resource
+from homeassistant.const import (
+    CONF_ADDRESS,
+    CONF_DEVICES,
+    CONF_DOMAIN,
+    CONF_ENTITIES,
+    CONF_ID,
+    CONF_NAME,
+    CONF_RESOURCE,
+    CONF_TYPE,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.typing import WebSocketGenerator
+
+DEVICES_PAYLOAD = {CONF_ID: 42, CONF_TYPE: "lcn/devices", "entry_id": ""}
+ENTITIES_PAYLOAD = {
+    CONF_ID: 42,
```

Comment:
Remove the ID, and use `client.send_json_auto_id(` in the tests so the message id does not need to be hard coded